{"name": "Проверка создание курьера, если создать пользователя с логином, который уже есть, возвращается ошибка.", "status": "passed", "start": 1747301335435, "stop": 1747301335535, "uuid": "c13e1226-015d-421d-bd25-254dd6e1f329", "historyId": "5ed32161bffc30105485bb86b08b2eb2", "testCaseId": "5ed32161bffc30105485bb86b08b2eb2", "fullName": "test.test_creat_courier.TestCreateCourier#test_create_courier_double_login", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_creat_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "GeoPC"}, {"name": "thread", "value": "9500-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_creat_courier"}]}